src/main/java/com/exasol/projectkeeper/validators/dependencies/ProjectDependencyReader.java

```diff
@@ -58,10 +60,10 @@ private Dependency convertPluginToDependency(final Plugin plugin) {
         return dependency;
     }
 
-    private ProjectDependency getDependenciesLicense(final Dependency dependency) {
+    private ProjectDependency getDependenciesLicense(final Dependency dependency, final MavenProject project) {
```

Comment:
Maybe sent repositories to this method instead of MavenProject?

Reply:
I like it like that, since it keeps the abstraction layer of `readDependencies` higher